app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: (ScihistDigicoll::Env.lookup(:no_reply_email_address) || "from@example.com")
+  default from: (ScihistDigicoll::Env.lookup(:digital_collections_email_address) || "from@example.com")
```

Comment:
As long as we're tweaking this stuff, lets get rid of the default `|| from@example.com` (we never want to send from there in production, if we accidentally did we might get penalized on spam lists and such?), and maybe change to `lookup!` which will raise  if no value is supplied.  Raise on app boot, since that's when this config code is executed.

Reply:
OK, great.